yb-voyager/cmd/migration_complexity.go

```diff
@@ -254,7 +254,7 @@ const explainTemplateHTML = `
 </p>
 `
 
-const explainTemplateText = `Reasoning: {{ .ComplexityRationale }}`
```

Comment:
nit: this is not a template string now, its a string `ComplexityRationale`, maybe we can avoid the whole template formating and execution for this.
So we can just return this `ComplexityRationale` in `buildMigrationComplexityExplanation` for this case

Reply:
yeah, will cleanup in further PRs